docs/process/release.md

```diff
@@ -128,6 +128,14 @@ git tag 'vXX'
 git push upstream && git push upstream --tags
 ```
 
+Double check that the packages we are about to publish are versioned correctly
+by running the following command.
+Their versions are derived from the git tag that we just created.
+
+```
+yarn admin release
```

Comment:
Should we run `yarn` before this to reload dependencies like the other commands? The user likely just checked out this branch and may have different dependencies loaded.

Reply:
The `release` script is really just used to check the package versions, so no yarn install is necessary.
I've renamed `release.ts` to `versions.ts`, so this should be less confusing.